### ISSUE TITLE
Ubuntu image is missing useradd

### ISSUE THREAD

**Issue by marverix** [Kong/docker-kong#675]
Ubuntu image is missing `useradd` command. In contrast Alpine image does `adduser`. In the git history one can see that Ubuntu image had also `useradd` in the past. Is it a bug or on purpose?

**Comment by chronolaw** [comment]
Yes, it is a good question. Could anyone in this team give us an answer?

**Comment by guoard** [comment]
I am almost sure that `kong` user is created by `.deb` package, so there is no need to create it manually in `Dockerfile`.
